examples/nlp/machine_translation/conf/aayn_base_megatron.yaml

```diff
@@ -44,7 +44,10 @@ model:
   src_language: 'en'
   tgt_language: 'de'
   max_generation_delta: 20 # Maximum decoder sequence length is encoder sequence length + this parameter.
-  micro_batch_size: 0 # This is just here to keep compute_consumed_samples() happy.
+  pretrained_model_path: null # Path to a pretrained model 
+  pretrained_model_type: T5
+  global_batch_size: 32 # This is just here to keep compute_consumed_samples() happy.
+  micro_batch_size: 32 # This is just here to keep compute_consumed_samples() happy.
```

Comment:
For now this is just a place-holder with tarred datasets, but once we move to megatron-based dataloaders, this will actually start to do something and is how you will configure grad acc etc.

Reply:
ok